docs/pages/admin-guides/management/guides/migrating-iam-ic-from-okta-to-teleport.mdx

```diff
@@ -0,0 +1,334 @@
+the integration's plugin resource with `tctl`.
+
+```console
+$ tctl edit plugins/aws-identity-center
+```
+
+The plugin resource is a YAML document that looks something like this:
+
+```YAML
+kind: plugin
+version: v1
+metadata:
+  labels:
+    teleport.dev/hosted-plugin: "true"
+  name: aws-identity-center
+spec:
+  Settings:
+    aws_ic:
+      # Account import filters. An absent or empty list of filters implies "manage all AWS accounts"
+      aws_accounts_filters:
+      - Include:
+          id: "637423191929"
+      - Include:
+          id: "730335414865"
+      - Include:
+          id: "058264527036"
+      - Include:
+          name_regex: ^Staging-.*$
+
+      # User provisioning filters. An absent or empty list of filters implies "provision all users to AWS"
+      user_sync_filters:
+      - labels:
+          teleport.dev/origin: okta
+
+      # Group import filters. See notes below. 
+      group_sync_filters:
+      - Include:
+          name_regex: '^Group #00\d+$'
+
+      access_list_default_owners:
+      - admin
+      arn: arn:aws:sso:::instance/ssoins-722326ecc902a06a
+      credentials_source: 2
+      integration_name: aws-identity-center
+      provisioning_spec:
+        base_url: https://scim.us-east-1.amazonaws.com/f3v9c6bc2ca-b104-4571-b669-f2eba522efe8/scim/v2
+      region: us-east-1
+```
+
+You can add or remove filters to the various filter sets. Once you save and quit
+the editor, `tctl` will replace the existing resource with your updated version.
+This will automatically restart the Identity Center Integration with the new
+filters.
+
+<Admonition type="note">
+As of Teleport 17.3, the `group_sync_filters` are only applied during the initial
+import at installation time, and there is no user accessible method of re-triggering
+the import. The only way to apply new group filters is to uninstall and re-install
+the plugin. 
+
+A later release of Teleport will re-trigger a group import when it detects a
+change in the group import filters.
+</Admonition>
+
+### Uninstall/reinstall
+
+A second alternative is to [uninstall](#deleting-the-identity-center-integration)
+and re-install the integration with the new parameters. Be warned that this may 
+result in user access disruptions while the integration is offline.
+
+## Step 7/7. Retire Okta group provisioning
+
+Once you are satisfied that an AWS Identity Center group has been migrated to 
+Teleport control, you can remove the corresponding push Group from the Okta
+Identity Center integration.
+
+## Deleting the Identity Center integration
+
+If you decide not to switch over to Teleport you can delete the Identity Center
+integration in two ways.
+
+Deleting the integration will automatically remove all of the Teleport resources 
+used by the integration to manage its own state, as well as all of the Teleport
+roles created by the integration used to grant AWS Identity Center account
+assignments and Access Lists imported from AWS Identity Center Groups.
```

Comment:
Is this true? Do we actually remove all imported access lists as well? I think we keep them for Okta integration, right? Just making sure this is accurate (and does the UI warn user about what will be removed? it definitely should cc @flyinghermit)

Also, what happens with access lists with IC-assigned-roles you created yourself that get pushed to IC? Those aren't removed, correct? This should be clarified here as well.

Reply:
Can confirm that this is the current behaviour. No warnings in `tctl plugins delete` at the moment.

Reply:
User created roles and access lists will not be deleted. If a user created access list refers to a plugin-created role, it will be left with a dangling role reference once the integration is deleted, and will need manual cleanup.

I have a bunch of `// TODO:`s in various places for walking User and Access List role assignments and clean them up when deleting plugin-created roles, but nothing actually implemented for it. 

Will add some notes about the above to the doc.

Reply:
Can you create a ticket to add confirmation/warning to tctl plugins delete? Maybe @flyinghermit can address it at the same time we add a similar warning to the UI.

Reply:
Ticket added https://github.com/gravitational/teleport.e/issues/6165